app/src/main/res/layout/fragment_parking.xml

```diff
@@ -0,0 +1,68 @@
+<?xml version="1.0" encoding="utf-8"?>
+<androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="wrap_content"
+    android:layout_height="wrap_content"
+    android:paddingBottom="@dimen/fragment_margin_bottom"
+    android:background="@color/primary_1"
+    tools:context=".activity.MainActivity">
+
+    <TextView
+        android:id="@+id/fragment_dialog_set_available_text"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:layout_margin="@dimen/activity_main_button_margin_top"
+        android:text="@string/fragment_dialog_set_available_text"
+        android:textStyle="bold"
+        android:textColor="@color/white"
+        app:layout_constraintBottom_toTopOf="@+id/fragment_dialog_set_available_number"
+        app:layout_constraintEnd_toEndOf="parent"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toTopOf="parent" />
+
+    <EditText
+        android:id="@+id/fragment_dialog_set_available_number"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:autofillHints=""
+        android:ems="5"
```

Comment:
Please, don't use harcodes. Why do you prefer to use ems instead of android:textSize, for example?

Reply:
Donde

Reply:
Done

Reply:
Already talked and solved.

Reply:
You can use integer https://stackoverflow.com/questions/19297522/android-integer-from-xml-resource

Reply:
please use an integer resource here, do not hardcode

Reply:
done